service/src/LevelZero.cpp

```diff
@@ -82,8 +82,8 @@ namespace geopm
                 if (num_subdevice == 0) {
                     std::cerr << "LevelZero::" << std::string(__func__)
                               << ": GEOPM Requires at least one subdevice. "
-                                 "Please check ZE_AFFINITY_MASK environment variable "
-                                 "setting.  Forcing device to act as sub-device" << std::endl;
+                              "Please check ZE_AFFINITY_MASK environment variable "
+                              "setting.  Forcing device to act as sub-device" << std::endl;
```

Comment:
Why were these whitespace changes made to the std:err strings?  astyle?
If there's no reason for this, please put them back the way they were.  Please do this for all similar std:err string whitespace changes.